index.js

```diff
@@ -502,6 +506,455 @@
   }
   S.is = def('is', {}, [TypeRep(a), $.Any, $.Boolean], is);
 
+  //. ### Showable
+
+  //# toString :: Any -> String
+  //.
+  //. Alias of [`Z.toString`][].
+  //.
+  //. ```javascript
+  //. > S.toString(-0)
+  //. '-0'
+  //.
+  //. > S.toString(['foo', 'bar', 'baz'])
+  //. '["foo", "bar", "baz"]'
+  //.
+  //. > S.toString({x: 1, y: 2, z: 3})
+  //. '{"x": 1, "y": 2, "z": 3}'
+  //.
+  //. > S.toString(S.Left(S.Right(S.Just(S.Nothing))))
+  //. 'Left(Right(Just(Nothing)))'
+  //. ```
+  S.toString = def('toString', {}, [$.Any, $.String], Z.toString);
+
+  //. ### Fantasy Land
+  //.
+  //. Sanctuary is compatible with the [Fantasy Land specification][FL].
+
+  //# equals :: a -> b -> Boolean
+  //.
+  //. Curried version of [`Z.equals`][].
+  //.
+  //. ```javascript
+  //. > S.equals(0, -0)
+  //. false
+  //.
+  //. > S.equals(NaN, NaN)
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 3]))
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just(['1', '2', '3']))
+  //. false
+  //. ```
+  S.equals = def('equals', {}, [a, b, $.Boolean], Z.equals);
```

Comment:
I see value in providing a second function. We might want to name it something bland like `equals_`, though, to avoid confusion with strict (`===`) equality.